src/Core/Translation/Storage/Provider/ModuleCatalogueLayersProvider.php

```diff
@@ -171,7 +188,16 @@ public function getDefaultCatalogue(string $locale): MessageCatalogue
     public function getFileTranslatedCatalogue(string $locale): MessageCatalogue
     {
         try {
-            return $this->getFileTranslatedCatalogueFinder()->getCatalogue($locale);
+            $moduleFilesCatalogue = $this->getBuiltInFileTranslatedCatalogueFinder()->getCatalogue($locale);
```

Comment:
You can use an early return here
```suggestion
            return $this->getBuiltInFileTranslatedCatalogueFinder()->getCatalogue($locale);
```

Reply:
? Maybe this comment is outdated. Or I dont understand it

Reply:
Outdated